src/structures/Sticker.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
 
 const Base = require('./Base');
-const { StickerFormatTypes } = require('../util/Constants');
+const { StickerFormatTypes, StickerTypes } = require('../util/Constants');
 const SnowflakeUtil = require('../util/SnowflakeUtil');
 
 /**
  * Represents a Sticker.
  * @extends {Base}
  */
 class Sticker extends Base {
+  /**
+   * @param {Client} client The instantiating client
+   * @param {Object} sticker The data for the sticker
```

Comment:
should there be a jsdoc type for the sticker data? (like what was introduced in #5862)